### PR TITLE
#171206860 allow users to upload travel documents

### PR DESCRIPTION
**Pull request by nshaphan**
#### What does this PR do?
Allow users to upload travel documents

#### Description of Task to be completed?
- [x] allow requester/travel admin to view all documents
- [x] allow the requester to add, remove document
- [x] allow travel admin to verify the document

#### How should this be manually tested?
- Checkout into `ft-upload-travel-documents-171304386` branch on backend
- Run `npm run watch`
- Checkout into `ft-upload-travel-documents-171206860` branch on fronted
- Run `npm run watch`
- Login as `requester user` on `account` menu click `travel documents`
image
- Login as `travel admin` click on `travel documents` menu

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171206860](https://www.pivotaltracker.com/story/show/171206860)

#### Screenshots (if appropriate)
image

image

image

#### Questions:
N/A

### CODE REVIEW

File: src/components/profile/EditProfile.js [hunk 1]
```diff
@@ -20,17 +20,17 @@ class EditProfile extends Component {
 		props.clearErrors();
 	}
 
+	componentWillUnmount() {
+		this.setState();
+	}
+
```
Comment (b0nbon1) [inline src/components/profile/EditProfile.js]: hey @nshaphan I think this line was to be removed

**Comment by kipropbrian** [comment]
Did you figure out how to view PDF in new tab

**Comment by nshaphan** [comment]
> Did you figure out how to view PDF in new tab

Yeah, I found out that `<embed />` can be used to render other external files such as `HTML, PDF, images, etc`

File: src/components/profile/EditProfile.js (hunk 1 above)
Comment (nshaphan) [reply src/components/profile/EditProfile.js]: Oh yeah, thanks @b0nbon1, let me work on it

File: src/components/TravelDocuments.js [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+							<div className='form-group'>
+								<div className='custom-file'>
+									<input
+										type='file'
+										className='custom-file-input'
+										id='customFile'
+										name='document'
+										onChange={e => handleFileUpload(e)}
+										required
+										data-testid='document'
```
Comment (RaymondMwaura) [inline src/components/TravelDocuments.js]: Add an 'accept' attribute here to act as validation on the file types a user can upload, for example:
`accept='image/png, image/jpeg, image/jpg, application/pdf'`

Comment (nshaphan) [reply src/components/TravelDocuments.js]: Thanks @RaymondMwaura let me look at that

Comment (nshaphan) [reply src/components/TravelDocuments.js]: > Add an 'accept' attribute here to act as validation on the file types a user can upload, for example:
> `accept='image/png, image/jpeg, image/jpg, application/pdf'`

Thanks, @RaymondMwaura, will greatly improve user experience

**Comment by RaymondMwaura** [comment]
Fix conflicts so that it can be merged.

**Pull request merged**